### ISSUE TITLE
Connecting to local network server

### ISSUE THREAD

**Issue by bugbountyguy** [jsknnr/enshrouded-server#29]
Hi,

I've been running into various little problems with this setup and am not quite sure what is going on. 

I've got a local network server I'm trying to setup like I've done for many other games. I started with custom port numbers, but when issues came up, I decided to try the default ports... It seems I still have the exact same issues.

What I'm running into is the following scenario: I've got a local network ip address for the server I'm setting up (let's just say 192.168.1.15 for this example). The firewall rules on the server have been approved for local network traffic to connect to both of the ports used by enshrouded. Using the default ports, I can favorite the game's server fine in steam as it sees the server. So if I put `192.168.1.15:15637` in steam favorites, it shows the latency + 0/16 players connected. When clicking connect, I put in the password, and click join.

From here, it goes to the game's main menu... it didn't connect. so then I click 'start' and I see my server at the very top of the list when clicking to join servers. I click join on it, and paste the password in again. 

From here, I can see clearly on my server the firewall accepting the incoming traffic to 15637. However, 15636 never comes in... ever. I even used wireshark on my computer to see and it never even attempts to reach that port on my server, only 15637. With that being said, it sits on a loading screen indefinitely, doing absolutely nothing - it's just stuck. 

Has anyone had any luck getting a local network connection to work on this game? I see these in my logs:
```
[online] Session accepted with peer ( id [my steam id] ).
[online] Added Peer #0.
[online] Client '[my steam id]' authenticated by steam
```
But nothing happens after that - it just deadends from there until I kill the game by clicking 'stop' in the steam client (no disconnect shows either).

note: This is just problem 1 of 2 I have, the 2nd is related to 'custom ports', but I want to at least see if I can get a 'normal' connection working first before I tackle that problem next. :)

**Comment by cp-fabian-pittroff** [comment]
I had the same issue. @jsknnr just update the jq command to this `jq --argjson g "$GAME_PORT" ...`. That removes the unwanted quotation on the ports and slot environments.

**Comment by jsknnr** [comment]
Nice find. I will make the change.

**Comment by jsknnr** [comment]
This has been fixed in image v2.0.2
